.changeset/early-beans-switch.md

```diff
@@ -0,0 +1,16 @@
+---
+"@fluidframework/driver-utils": major
+"@fluidframework/local-driver": major
+"@fluidframework/odsp-driver": major
+"@fluidframework/odsp-urlresolver": major
+"@fluid-experimental/property-dds": major
+"@fluidframework/routerlicious-driver": major
+"@fluidframework/routerlicious-urlresolver": major
+"@fluid-private/test-end-to-end-tests": major
+"@fluidframework/test-utils": major
+"@fluidframework/tinylicious-driver": major
+---
+
+Resolved URLs no longer use non-standard protocols
+
+Previously, IResolvedUrl.url could use a non-standard protocol like fluid://, fluid-odsp://, or fluid-test://. These have been replaced with https:// to permit standards-compliant URL parsing.
```

Comment:
Would API council (yes I know that includes you) have opinions here? Do we care that these URIs won't be addressable on the internet?